plugins/LilyPad/Config.cpp

```diff
@@ -48,7 +48,7 @@ void PCSX2_overrideConfig(GeneralConfig &config_in_out)
     config_in_out.disableScreenSaver = 0;   // Not required - handled internally by PCSX2
     config_in_out.escapeFullscreenHack = 0; // Not required - handled internally by PCSX2
     config_in_out.saveStateTitle = 0;       // Not required - handled internally by PCSX2
-    config_in_out.closeHacks = 0;           // Cannot function when used by PCSX2
+    config_in_out.closeHack = 0;            // Cannot function when used by PCSX2
```

Comment:
Is the remaining close hack useful? I'm not sure ExitProcess() allows for proper cleanup to happen.